scripts/degenApe.js

```diff
@@ -37,72 +37,31 @@ const allReports = [];
 // Addresses & Contracts
 const governance = "0x4204FDD868FFe0e62F57e6A626F8C9530F7d5AD1";
 const strategist = "0x4204FDD868FFe0e62F57e6A626F8C9530F7d5AD1";
-const controller = "0xdc954e7399e9ADA2661cdddb8D4C19c19E070A8E";
+const controller = "0xe5E231De20C68AabB8D669f87971aE57E2AbF680";
 const timelock = "0x4204FDD868FFe0e62F57e6A626F8C9530F7d5AD1";
 const harvester = ["0x0f571D2625b503BB7C1d2b5655b483a2Fa696fEf"];
 
 const contracts = [
-  // "src/strategies/near/trisolaris/strategoy-tri-$aurora-$near-lp.sol:StrategyTriAuroraNearLp",
-  "src/strategies/near/trisolaris/strategy-tri-near-usdc-lp.sol:StrategyTriNearUsdcLp",
-  // "src/strategies/near/trisolaris/strategy-tri-$near-$usdt-lp.sol:StrategyTriNearUsdtLp"
+  "src/strategies/gnosis/curve/strategy-curve-3pool-lp.sol:StrategyXdaiCurve3CRV"
 ];
 
 const testedStrategies = [
-  "0xE197b88C0C94F6396a66f964dbd6F87F11EF95D4"
 ];
 
-// Functions
-const sleep = async (ms, active = true) => {
-  if (active) {
-    console.log("Sleeping...")
-    return new Promise((resolve) => setTimeout(resolve, ms));
-  }
-};
-
-// Use this for verification of 5 contracts or less
-const fastVerifyContracts = async (strategies) => {
-  // await exec("npx hardhat clean");
-  console.log(`Verifying contracts...`);
-  await Promise.all(strategies.map(async (strategy) => {
-    try {
-      await hre.run("verify:verify", {
-        address: strategy,
-        constructorArguments: [governance, strategist, controller, timelock],
-      });
-    } catch (e) {
-      console.error(e);
-    }
-  }));
-}
-
-// Use this for verificationof 5 contracts or more
-const slowVerifyContracts = async (strategies) => {
-  for (strategy of strategies) {
-    try {
-      await hre.run("verify:verify", {
-        address: strategy,
-        constructorArguments: [governance, strategist, controller, timelock],
-      });
-    } catch (e) {
-      console.error(e);
-    }
-  }
-}
-
 const executeTx = async (calls, tx, fn, ...args) => {
-  await sleep(sleepTime, sleepToggle);
-  // if (!txRefs[tx]) { recall(executeTx, calls, tx, fn, ...args) }
+  let transaction;
+  await sleep(sleepConfigs);
   try {
-    if (!txRefs[tx]) {
-      txRefs[tx] = await fn(...args)
+    if (!txRefs.tx) {
+      transaction = await fn(...args)
       if (tx === 'strategy') {
-        await txRefs[tx].deployTransaction.wait();
+        await transaction.deployTransaction.wait();
       }
       else if (tx === 'jar') {
-        const jarTx = await txRefs[tx].deployTransaction.wait();
-        txRefs['jarStartBlock'] = jarTx.blockNumber;
+        const jarTx = await transaction.deployTransaction.wait();
+        txRefs.'jarStartBlock' = jarTx.blockNumber;
       } else {
-        await txRefs[tx].wait();
+        await transaction.wait();
       }
     }
   } catch (e) {
```

Comment:
I can't leave a comment on line 73, `looks like something is broken`, but shouldn't this function THROW the error so the script knows to stop or something? You are swallowing the error here. If something is broken, your code will just blindly continue along trying to execute instead of abort or something.